README.md

```diff
@@ -39,6 +39,28 @@ It's a [WebAssembly](https://webassembly.org/) build of the VM compiled using [e
 Note that in the webassembly version of the language, some features (input, relative import, etc.) have disabled, has
 limited memory allocations, and the IO calls might be slower.
 
+### On localhost
```

Comment:
The heading should contain only 2 hashes `##` to match other ones. And the title should be changed to something like `On Your System` because `On Localhost` seems like it's a web-based application. feel free to come up with anything you like.